File: src/main/scala/li/cil/oc/integration/cofh/tileentity/ModCoFHTileEntity.java
package li.cil.oc.integration.cofh.tileentity;

import li.cil.oc.api.Driver;
import li.cil.oc.integration.IMod;
import li.cil.oc.integration.Mods;

public final class ModCoFHTileEntity implements IMod {
    @Override
    public Mods.Mod getMod() {
        return Mods.CoFHTileEntity();
    }

    @Override
    public void initialize() {
        Driver.add(new DriverEnergyInfo());
        Driver.add(new DriverRedstoneControl());
        Driver.add(new DriverSecureTile());
    }
}

File: src/main/scala/li/cil/oc/integration/cofh/tileentity/ModCoFHTileEntity.scala
package li.cil.oc.integration.cofh.tileentity

import li.cil.oc.api.Driver
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModCoFHTileEntity extends ModProxy {
  override def getMod = Mods.CoFHCore

  override def initialize() {
    Driver.add(new DriverEnergyInfo)
    Driver.add(new DriverRedstoneControl)
    Driver.add(new DriverSecureTile)
    Driver.add(new DriverSteamInfo)
  }
}